src/main/webapp/index.html

```diff
@@ -105,7 +104,7 @@ <h4>Learn to code</h4>
           <a href="https://bit.ly/3i4nZv8" target="_blank">geeksforgeeks: website that teaches you programer with examples and videos</a>
```

Comment:
GeeksforGeeks: A Website that ...
That's how they capitalize it on the site, and then just capitalizing the start of the description